packages/@okta/vuepress-site/docs/guides/customize-themes/index.md

```diff
@@ -32,30 +35,17 @@ The [Brands API](/docs/reference/api/brands/) allows you to set all of the follo
 * Favicon
 * Background image
 
-## Before you begin
-
-You need to make sure that you have an [Okta organization](/docs/guides/quickstart/cli/main/#create-your-okta-organization) set up to test this functionality, and you need to request access to the Brands API from [Okta support](https://support.okta.com/help).
-
-It is up to you how you make requests to the API to update your branding. In this guide, we are demonstrating the required API calls using a Postman collection to demonstrate them in a language/platform neutral way. To run the API calls:
-
-1. [Create an API token](/docs/guides/create-an-api-token/) to use when accessing the API.
-1. [Download](https://www.postman.com/downloads/) and install Postman.
-1. After you install Postman, import the Okta environment and add your Okta domain and API token to Postman, as explained in [Use Postman with the Okta REST APIs > Set up your environment](/code/rest/#set-up-your-environment).
-1. Click **Run in Postman** to add the Brands collection to Postman, which allows you to test the API calls that are described below.
-
-[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/1d58ab8a3909dd6a3cfb)
-
 ## Important: Overriding themes in the code template editors
 
-You can customize individual parts of your Okta org experience using the various code editors we've made available in the Okta Admin Console (for example, **Settings** > **Customizations** > **Custom Sign-in** to customize the Okta-hosted sign-in page). Note that when you use the Brands API, your custom brand settings won't apply to the places where you've customized the theme settings.
+You can customize individual parts of your Okta org experience using the various code editors available in the Admin Console (for example, **Settings** > **Customizations** > **Custom Sign-in** to customize the Okta-hosted sign-in page). However, when you use the Brands API, your custom brand settings won't apply to theme settings that you customized.
```

Comment:
**Customizations** > **Sign-in page code editor**

"When you use the Brands API, your custom brand settings may be overwritten if your custom code deletes the classes and variables used by the Branding theme objects."

Reply:
product docs actually lists all the variables https://help.okta.com/oie/en-us/Content/Topics/Settings/branding-variables.htm so if we want to be super clear its "if you delete or augment how these variables are used in your custom code, then the Branding features will be overwritten by your custom code."